sdk/bpf/c/inc/solana_sdk.h

```diff
@@ -526,6 +526,154 @@ uint64_t sol_try_find_program_address(
     uint8_t *bump_seed
 );
 
+/**
+ * BigNum sol_bignum_from_u32
+ *
+ * @param out_bn_ffi_ptr address of vector FfiBigNumber result structure (r/w)
+ * @param val_u32 unsigned 32 bit value to assign to the new object
+ */
+static uint64_t sol_bignum_from_u32(
+    const uint64_t *out_bn_ffi_ptr,
```

Comment:
Please add the proper struct here for C programs.

Reply:
Added in 2289d94cbed61cf1250360cef0556a87e15c07cc